src/datastore/postgres-store.ts

```diff
@@ -5482,15 +5443,15 @@ export class PgDataStore
               AND type_id = 0
               AND token_transfer_recipient_address = $1
           ) transfers
-        ${whereClause}
+        WHERE block_height <= $5
```

Comment:
It seems like this ignores the `whereClause` logic above and doesn't support the `atSingleBlock: false` condition? There should be a unit test that would have caught this.